docs/ecs-compose-features.md

```diff
@@ -0,0 +1,165 @@
+|                                |   |
+| __Config__                     | x |
+|                                |   |
+
+
+## Logs
+
+Application logs can be obtained container with `docker compose logs`.
+The Docker ECS integration relies on AWS CloudWatch Logs to collect logs from all containers. CloudWatch can be customized by setting service `logging.driver_opts`
+by passing configuration attributes prefixed with `awslogs-`.
+
+```yaml
+  test:
+    image: mycompany/webapp
+    logging:
+      driver-opts:
+        awslogs-datetime-pattern: "some-pattern"
+```
+
+
+## Exposing ports
+
+When one or more services expose ports, a Load Balancer is created for the application.
+As all services are expose through the same Load Balancer, only one service can expose a given port number.
+The source and target ports defined in the Compose file MUST be the same, as service-to-service communication don't go trought the Load Balancer and could not
+benefit from Listeners abstraction to assign a distinct published port. 
+
+If services in the Compose file only expose ports 80 or 443, an Application Load Balancer is created, otherwise ECS integration will provision a Network Load Balancer.
+HTTP services using distinct ports can force use of an ALB by claiming the http protocol with `x-aws-protocol` custom extension within the port declaration:
+
+```yaml
+  test:
+    image: mycompany/webapp
+    ports:
+      - target: 8080
+        x-aws-protocol: http
+
+```
+
+## Persistent volumes
+
+Docker volumes are mapped to EFS file systems. Volumes can be external (`name` must then be set to filesystem ID) or will be created when the application is
+first deployed. `docker compose down` will NOT delete the filesystem, and it will be re-attached to the application on future runs.
+`driver_opts` can be used to tweak the EFS filsystem.
+
+Volume mount can be customized to workaround Posix filesystem permission issues by setting user and group IDs to be used to write to filesystem, whatever user
+is configured to run the container.
+
+```yaml
+services:
+    myservice:
+        image: mycompany/webapp
+        volumes:
+        - mydata:/mount/testvolumes
+
+volumes:
+  mydata:
+    driver_opts:
+      performance-mode: maxIO 
+      throughput-mode: bursting
+      uid: 0
+      gid: 0
+```
+
+
+## Secrets
+
+Secrets can be defined in compose files, and will need secret files available at deploy time next to the compose file.
+The content of the secret file will be made available inside selected containers, by default under `/run/secrets/<SECRET_NAME>`.
+External secrets are also supported, `name` must then be set to secret's ARN
+
+```yaml
+services:
+    nginx:
+        image: mycompany/webapp
+        secrets:
+          - mysecret
+
+secrets:
+  mysecret:
+    file: ./my_secret1.txt
+```
+
+
+## Container Resources
+
+CPU and memory reservations and limits can be set in compose.
```

Comment:
Even if it's standard compose file format, having an example here would be useful (or a link users can click to see compose file reference on CPU/memory limits and some example)